x-pack/plugins/cross_cluster_replication/public/app/sections/home/home.js

```diff
@@ -20,90 +20,154 @@ import {
   EuiPageBody,
   EuiPageContent,
   EuiSpacer,
+  EuiTab,
+  EuiTabs,
   EuiText,
   EuiTitle,
 } from '@elastic/eui';
 
 import { listBreadcrumb } from '../../services/breadcrumbs';
 import routing from '../../services/routing';
 import { AutoFollowPatternList } from './auto_follow_pattern_list';
+import { FollowerIndicesList } from './follower_indices_list';
 import { SectionUnauthorized } from '../../components';
 
 export const CrossClusterReplicationHome = injectI18n(
   class extends PureComponent {
     static propTypes = {
       autoFollowPatterns: PropTypes.array,
+      isAutoFollowApiAuthorized: PropTypes.bool,
+      followerIndices: PropTypes.array,
+      isFollowerIndexApiAuthorized: PropTypes.bool,
     }
 
     state = {
-      sectionActive: 'auto-follow'
+      activeSection: 'follower_indices'
     }
 
+    tabs = [{
+      id: 'follower_indices',
+      name: (
+        <FormattedMessage
+          id="xpack.crossClusterReplication.autoFollowPatternList.followerIndicesTitle"
+          defaultMessage="Follower indices"
+        />
+      )
+    }, {
+      id: 'auto_follow_patterns',
+      name: (
+        <FormattedMessage
+          id="xpack.crossClusterReplication.autoFollowPatternList.autoFollowPatternsTitle"
+          defaultMessage="Auto-follow patterns"
+        />
+      )
+    }]
+
     componentDidMount() {
+      const { match: { params: { section } } } = this.props;
+      this.setState({
+        activeSection: section
+      });
       chrome.breadcrumbs.set([ MANAGEMENT_BREADCRUMB, listBreadcrumb ]);
     }
 
+    static getDerivedStateFromProps(props) {
+      const { match: { params: { section } } } = props;
+      return {
+        activeSection: section
+      };
+    }
+
+    onSectionChange = (section) => {
+      routing.navigate(`/${section}`);
```

Comment:
I don't like the way this routing is handled (along with `getDerivedStateFromProps` above for setting active tab when user hits back button). Current behavior reloads the entire page when a different tab is selected. Open to ideas!

Reply:
This is working very well on my side (tested on Chrome and Firefox). I don't get the page refresh.